libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1046,6 +1082,11 @@ unsigned CompilerUtils::loadFromMemoryHelper(Type const& _type, bool _fromCallda
 	return numBytes;
 }
 
+IntegerType const& CompilerUtils::fixedPointTypeAsIntegerType(FixedPointType const &_type)
```

Comment:
You cannot return a temporary value by reference, so this sthould be just `IntegerType`.

Reply:
And yes, C++ is a really shitty language for not auto-detecting this...

Reply:
I tried using a non reference but I got something about IntegerType not being copyable.